examples/decoding/cohy_decomposition_plotting.py

```diff
@@ -0,0 +1,170 @@
+# Background
+# ----------
+# Multivariate forms of signal analysis allow you to simultaneously consider the
+# activity of multiple signals. In the case of connectivity, the interaction between
+# multiple sensors can be analysed at once and the strongest components of this
+# interaction captured in a lower-dimensional set of connectivity spectra. This approach
+# brings not only practical benefits (e.g. easier interpretability of results from the
+# dimensionality reduction), but can also offer methodological improvements (e.g.
+# enhanced signal-to-noise ratio and reduced bias).
+#
+# Coherency-based methods are popular approaches for analysing connectivity, capturing
+# correlations between signals in the frequency domain. Various coherency-based
+# multivariate methods exist, including: canonical coherency (CaCoh; multivariate
+# measure of coherency/coherence) :footcite:`VidaurreEtAl2019`; and maximised imaginary
+# coherency (MIC; multivariate measure of the imaginary part of coherency)
+# :footcite:`EwaldEtAl2012`.
+#
+# These methods are described in detail in the following examples:
+#  - comparison of coherency-based methods - :doc:`../compare_coherency_methods`
+#  - CaCoh - :doc:`../cacoh`
+#  - MIC - :doc:`../mic_mim`
+#
+# The CaCoh and MIC methods work by finding spatial filters that decompose the data into
+# components of connectivity, and applying them to the data. Connectivity can then be
+# computed on this transformed data (see :doc:`cohy_decomposition` for more
+# information).
+#
+# However, in addition to the connectivity scores, useful insights about the data can be
+# gained by visualising the topographies of the spatial filters and their corresponding
+# spatial patterns. These provide important information about the spatial distributions
+# of connectivity information, and represent two complementary aspects:
+#
+# - The filters represent how the connectivity sources are extracted from the channel
+#   data, akin to an inverse model.
+# - The patterns represent how the channel data is formed by the connectivity sources,
+#   akin to a forward model.
+#
+# This distinction is discussed further in Haufe *et al.* (2014)
+# :footcite:`HaufeEtAl2014`, but in short: **the patterns should be used to interpret
+# the contribution of distinct brain regions/sensors to a given component of
+# connectivity**. Accordingly, keep in mind that the filters and patterns are not a
+# replacement for source reconstruction, as without this the patterns will still only
+# tell you about the spatial contributions of sensors, not underlying brain regions,
+# to connectivity.
+
+########################################################################################
+# Generating the filters and patterns
+# -----------------------------------
+# We will first load some example MEG data which we will generate the spatial filters
+# and patterns for, and divide it into epochs.
```

Comment:
Explain briefly what kind of data this is? What paradigm?

Reply:
I might be blind, but I didn't see the `fieldtrip_cmc` data described in the MNE docs with the other open datasets. From the name I'm guessing it has something to do with the data described here: https://www.fieldtriptoolbox.org/tutorial/coherence/

Updated to this:
https://github.com/mne-tools/mne-connectivity/blob/7709a329172c2e2e9b2f2048134ecacebe436ad3/examples/decoding/cohy_decomposition_plotting.py#L70-L72

Reply:
Perhaps it is worth adding a link to that specific tutorial to "see for more information"?

I always find it useful as a use to be able to conceptually tie w/e data we're looking at and the question to why we're using a specific computational method.

Reply:
@adam2392 Happy to include a link to the ft example. Just FYI there are a few differences between theirs and ours:
- we look at connectivity between MEG sensors using icoh
- they look at connectivity between MEG and EMG using coh

But it still explains the data a bit more so can add this

Reply:
Done!